Sources/Conduit/Auth/OAuth2RequestPipelineMiddleware.swift

```diff
@@ -149,59 +152,31 @@ public struct OAuth2RequestPipelineMiddleware: RequestPipelineMiddleware {
         completion(.value(request))
     }
 
-    private func buildTokenRefreshRequestFor(token: BearerToken, completion: Result<URLRequest>.Block) {
+    private func refresh(token: BearerToken, completion: @escaping Result<BearerToken>.Block) {
+        Auth.Migrator.notifyTokenPreFetchHooksWith(client: clientConfiguration,
+                                                   authorizationLevel: authorization.level)
         guard let refreshToken = token.refreshToken else {
             logger.warn([
                 "A request required Bearer authorization, but the expired token",
                 "does not have an available refresh token"
-            ].joined(separator: " "))
+                ].joined(separator: " "))
```

Comment:
Nit: could we update this to Swift 4 multiline strings? (""")

Reply:
I believe the intention of this was to reduce line length. A multiline string would add carriage returns